INSTALL-docker.md

```diff
@@ -0,0 +1,35 @@
+# Installing the CircleCI Runner on a host with Docker
+
+## Create a Dockerfile that extends the CircleCi Runner image
+
+Here we install python3 on top of the base image
+
+*Dockerfile.runner.extended*
+
+```
+FROM circleci/runner:launch-agent
+RUN apt-get update; \
+    apt-get install --no-install-recommends -y \
+        python3
+```
+
+## Build the Docker image
+
+```bash
+docker build --file ./Dockerfile.runner.extended .
+```
+
+## Start the Docker container
+
+```bash
+CIRCLECI_RESOURCE_CLASS=<resource-class> CIRCLECI_API_TOKEN=<runner-token> docker run --env CIRCLECI_API_TOKEN --env CIRCLECI_RESOURCE_CLASS --name <container-name> <container-id-from-previous-step>
```

Comment:
```suggestion
CIRCLECI_RESOURCE_CLASS=<resource-class> CIRCLECI_API_TOKEN=<runner-token> docker run --env CIRCLECI_API_TOKEN --env CIRCLECI_RESOURCE_CLASS --name <container-name> <image-id-from-previous-step>
```

Reply:
@ProjectFrank That's how I originally had it, but the docker command is visible in `docker ps` in some versions of docker client.